cheatsheets/Forgot_Password_Cheat_Sheet.md

```diff
@@ -1,60 +1,104 @@
-# Introduction
+# Forgot Password and MFA Cheat Sheet
 
-This article provides a simple model to follow when implementing a *forgot password* web application feature.
+## Introduction
 
-# The Problem
+In order to implement a proper user management system, almost all systems integrate a **Forgot Password/MFA?** service, which allows the user to request a reset to their password or MFA whenever they forget any of them, or if their account ever gets breached.
 
-There is no industry standard for implementing a **Forgot Password** feature. The result is that you see applications forcing users to jump through myriad hoops involving emails, special URLs, temporary passwords, personal security questions, and so on. In the end you have to reset it to a new value.
+As much as is this functionality looks straight-forward and easy to implement, the details of its implementation makes it a sweet spot for security attacks, such as the renowned [user enumeration attack](https://owasp.org/www-project-web-security-testing-guide/stable/4-Web_Application_Security_Testing/03-Identity_Management_Testing/04-Testing_for_Account_Enumeration_and_Guessable_User_Account.html).
```

Comment:
There's an extra word at the beginning: "is". This should instead read

> As much as this functionality...

Reply:
Nice suggestion :+1: